crates/viewer/re_viewer/src/app.rs

```diff
@@ -83,12 +83,8 @@ pub struct StartupOptions {
     /// This also can be changed in the viewer's option menu.
     pub video_decoder_hw_acceleration: Option<re_video::decode::DecodeHardwareAcceleration>,
 
-    /// Interaction between JS and timeline.
-    ///
-    /// This field isn't used directly, but is propagated to all recording configs
-    /// when they are created.
-    #[cfg(target_arch = "wasm32")]
-    pub timeline_options: Option<crate::web::TimelineOptions>,
+    /// Interaction between JS and the viewer.
```

Comment:
(you had a comment fix locally for this during our call today that hasn't been pushed yet)